docs/server/configuration.md

```diff
@@ -0,0 +1,68 @@
+# Server Configuration
+
+The configuration of the software is done in the client and [campaign settings](/docs/dm/settings/).
+While you actually could modify the configuration stored in the save file (see below), it is not recommended and, thus, not documented, here.
+In contrast, this section focuses on the configuration that should be done within the central configuration file, which is `./server/server_config.cfg`.
+
+The configuration file is divided up into two sections: Webserver, and General.
+These sections are indicated by these designations, within square brackets (e.g. `[Webserver]`).
+Within each section, there are certain variables defined via a name and their value, using the scheme `name = value`, followed by a line-break.
+A leading `#` denotes a comment; every line with a leading `#` will be ignored.
+
+Only change the configuration if something does not work accordingly and you know what you're doing.
+
+## Webserver
+
+This section contains several variables concerning the webserver that distributes the client to the players.
+
+### host
+`host` (active by default, default: `0.0.0.0`, accepting IP addresses) tells the server on what IP address it is reachable.
```

Comment:
It accepts any resolvable string, so also for example dns names, not only ip addresses